cmd/tetra/getevents/getevents.go

```diff
@@ -133,5 +139,6 @@ func New() *cobra.Command {
 	flags.Bool("host", false, "Get host events")
 	flags.Bool("timestamps", false, "Include timestamps in compact output")
 	viper.BindPFlags(flags)
+
```

Comment:
nit: white-space change.

Reply:
You would like this whitespace removed? Remove line 142?